### PR TITLE
server documentation (for issue #86)

### PR DESCRIPTION
**Pull request by mozkeeler**
Let me know if there's not enough detail or if anything is unclear.

### CODE REVIEW

File: doc/server.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  * Listening on 8443 turns into listening on 443 through AWS magic, AIUI
+  * Forwarding to port 8158 means we can run an unpriveleged node server and not have to enter our key's passphrase every time we restart that server
+3. Install things as the 'app' user on the AWS instance
+  * Node packages 'promise', 'http', 'sqlite3'
+  * server/server.js
+4. Start the servers:
+  * As ec2-user: 'sudo /etc/init.d/nginx start' (requires key passphrase, logs in /var/log/nginx)
+  * As app: 'node server.js' (this creates the file 'events.sqlite')
+5. Are events coming in as expected?
+  * Try 'sqlite3 events.sqlite', then 'select * from events where 1=1;'
```
Comment (monicachew) [inline doc/server.md]: can omit 1=1, right?

File: doc/server.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+               # redirect server error pages to the static page /50x.html
+               #
+               error_page   500 502 503 504  /50x.html;
+               location = /50x.html {
+                   root   /usr/share/nginx/html;
+               }
+            }
+  * cert.{pem,key} is the certificate and key for the server
+  * Listening on 8443 turns into listening on 443 through AWS magic, AIUI
+  * Forwarding to port 8158 means we can run an unpriveleged node server and not have to enter our key's passphrase every time we restart that server
```
Comment (monicachew) [inline doc/server.md]: unprivileged

File: doc/server.md (hunk 1 above)
Comment (mozkeeler) [inline doc/server.md]: Yeah, I guess the whole where clause can go away.

File: doc/server.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+               server_name  blushproof.org;
+
+               ssl                  on;
+               ssl_certificate      cert.pem;
+               ssl_certificate_key  cert.key;
+
+               ssl_session_timeout  5m;
+
+               ssl_protocols  TLSv1;
+               ssl_ciphers  HIGH:!aNULL:!MD5;
```
Comment (monicachew) [inline doc/server.md:22]: Can you put in a comment, no null-ciphers and no md5?

File: doc/server.md (hunk 2 above)
Comment (mozkeeler) [inline doc/server.md]: When I rule the world, I'm going to change the spelling of that word.

**Pull request merged**